tests/unit/mixins/oidc-end-session-route-mixin-test.js

```diff
@@ -0,0 +1,33 @@
+import EmberObject from "@ember/object";
+import OIDCEndSessionRouteMixin from "ember-simple-auth-oidc/mixins/oidc-end-session-route-mixin";
+import { module, test } from "qunit";
+import { setupTest } from "ember-qunit";
+import config from "ember-get-config";
+
+const { endSessionEndpoint, afterLogoutUri } = config["ember-simple-auth-oidc"];
+
+module("Unit | Mixin | oidc-end-session-route-mixin", function(hooks) {
+  setupTest(hooks);
+
+  test("it can make an invalidate request", function(assert) {
+    assert.expect(3);
+
+    const Route = EmberObject.extend(OIDCEndSessionRouteMixin);
+
+    const subject = Route.create({
+      session: EmberObject.create({
+        data: { authenticated: { id_token: "myIdToken" } },
+        async invalidate() {}
+      }),
+      _redirectToUrl(url) {
+        assert.ok(new RegExp(endSessionEndpoint).test(url));
+        assert.ok(
+          new RegExp(`post_logout_redirect_uri=${afterLogoutUri}`).test(url)
+        );
+        assert.ok(new RegExp("id_token_hint=myIdToken").test(url));
```

Comment:
How about using https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/String/includes instead of Regex?